src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -442,14 +442,18 @@ public function testChmodChangesFileMode()
         $this->assertFilePermissions(400, $file);
     }
 
-    public function testChmodWrongMod()
+    public function testChmodWithWrongModLeavesPreviousPermissionsUntouched()
     {
         $this->markAsSkippedIfChmodIsMissing();
 
         $dir = $this->workspace.DIRECTORY_SEPARATOR.'file';
         touch($dir);
 
+        $permissions = fileperms($dir);
+
         $this->filesystem->chmod($dir, 'Wrongmode');
+
+        $this->assertSame($permissions, fileperms($dir));
```

Comment:
Looking at the build job it seems that this is at least not the same behaviour with HHVM and PHP. So what do we do here?